### PR TITLE
fixed after bumping Rightmesh to version 5.0

### PR DESCRIPTION
**Pull request by ghost**
- Changed MeshID to MeshId
- fixed some random crashes
- changed variable names

### CODE REVIEW

**Review by fractalic — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -62,22 +62,20 @@ android {
             minifyEnabled false
         }
     }
-
-    // Allows the use of lambda and method reference syntax.
+     // Allows the use of lambda and method reference syntax.
```
Comment (FrazerSeymour) [inline app/build.gradle:65]: We don't actually need this whole block anymore.

**Comment by FrazerSeymour** [comment]
Did you run this through Checkstyle?

File: app/build.gradle (hunk 1 above)
Comment (ghost) [reply app/build.gradle:65]: we are still using lamdas in Ripple, without the block it gives an error

Comment (FrazerSeymour) [reply app/build.gradle:65]: Ah cool. Sorry, thought that went away when Android started supporting Java 8. Guess we just don't need RetroLambda anymore, but looks like that is gone.

**Review by FrazerSeymour — APPROVED** [review]

**Pull request merged**